config/samples/registry/goharbor_v1alpha2_registry.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: goharbor.io/v1alpha2
+kind: Registry
+metadata:
+  name: sample
+spec:
```

Comment:
`make sample` currently doesn't work unless we add the spec replicas: 1.
Error message:
> The Registry "sample-sample" is invalid: .spec.replicas: Invalid value: 0: .spec.replicas accessor error: 1 is of the type float64, expected int64

Reply:
I then encounter the following error:
> Error from server (InternalError): error when creating "STDIN": Internal error occurred: failed calling webhook "vregistry.kb.io": Post https://harbor-operator-webhook-service.harbor-operator-system.svc:443/validate-goharbor-io-v1alpha2-registry?timeout=30s: no endpoints available for service "harbor-operator-webhook-service"

Reply:
Webhook error by running the following commands:
```
kubectl delete MutatingWebhookConfiguration harbor-operator-mutating-webhook-configuration
kubectl delete ValidatingWebhookConfiguration harbor-operator-validating-webhook-configuration
```

Those webhooks were probably deployed by running `make deploy`.

Would it be safe to include these commands to the `make sample` target?